static/js/hoc/withChannelSidebar.js

```diff
@@ -4,25 +4,27 @@ import R from "ramda"
 
 import ChannelSidebar from "../components/ChannelSidebar"
 import Sidebar from "../components/Sidebar"
-import { Grid, Cell } from "../components/Grid"
+import withTwoColumns from "./withTwoColumns"
 
 const withChannelSidebar = R.curry(
   (className: string, WrappedComponent: Class<React.Component<*, *>>) => {
+    const TwoColumnComponent = withTwoColumns(className, WrappedComponent)
```

Comment:
IMO this is getting a bit too difficult to trace the code, when we have a HOC that ends up wrapping the component with another HOC. If I want to figure out where the class name (`main-content two-column channel-page`) set on the channel page container, for instance, is set I have to open and look at three different files. what do you think about just keeping things simple with a `Grid` and two `Cells` in this component?

Reply:
Sounds good